pages/instructorJobSequence/instructorJobSequence.ejs

```diff
@@ -2,7 +2,7 @@
 <html>
   <head>
     <%- include('../partials/head'); %>
-    <script src="/javascripts/socket.io.js"></script>
+    <script src="/node_modules/socket.io-client/dist/socket.io.slim.js"></script>
```

Comment:
Awesome! Can we delete `/javascripts/socket.io.js`?

Reply:
It's still in use by a few pages, but I'll replace it with the new script provided by `node_modules`. I'll assume there aren't any courses still using this.